src/Employer/UnitTests/Recruit.Vacancies.Client/Application/Rules/VacancyRules/VacancyProfanityChecksRuleTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Application.Rules.BaseRules;
+using Esfa.Recruit.Vacancies.Client.Application.Rules.VacancyRules;
+using Esfa.Recruit.Vacancies.Client.Application.Services;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Xunit;
+
+namespace Esfa.Recruit.Employer.UnitTests.Recruit.Vacancies.Client.Application.Rules.VacancyRules
```

Comment:
Could you either create a more specific namespace or chop off the `Esfa.Recruit.Employer.UnitTests.` or call the namespace `Recruit.Vacancies.Client.UnitTests.Application.Rules.VacancyRules`

Reply:
Moved the class to the correct location